scripts/packaging/generate_asset_cache.py

```diff
@@ -37,9 +37,8 @@
     raise RuntimeError('Error, WEBOTS_HOME variable is not set.')
 
 
-def generate_asset_cache(tag):
-    with open(os.path.join(WEBOTS_HOME, 'resources', 'version.txt'), 'r') as file:
-        folder_name = f'assets-{file.readline().strip()}'
+def generate_asset_cache(tag, version_name):
```

Comment:
This function is also used at line 79 in this file with just one argument.
It should be fixed accordingly.